component/loki/source/windowsevent/component.go

```diff
@@ -0,0 +1,37 @@
+//go:build !windows
```

Comment:
nit: Can we rename this file component_stub? I would generally see `component.go` and think it's where the main implementation is, so calling it out as a stub would be slightly less confusing.